packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { createLogger } from '@stoplight/prism-core';
+import fastify = require('fastify');
+import { relative, resolve } from 'path';
+import { createServer } from '../';
+import { IPrismHttpServer } from '../types';
+
+const logger = createLogger('TEST', { enabled: false });
+
+async function instantiatePrism(fixture: string) {
+  const server = createServer({}, { components: { logger }, config: { mock: { dynamic: false } } });
+  await server.prism.load({
+    path: relative(process.cwd(), resolve(__dirname, 'fixtures', fixture)),
+  });
+  return server;
+}
+
+const expectPayload = (response: fastify.HTTPInjectResponse) => {
+  const parsed = JSON.parse(response.payload);
```

Comment:
seems like `expectPayload` only depends on `response.payload`, so there's no need to pass the whole `response` object most likely